src/EventStoreWorker.php

```diff
@@ -48,28 +51,44 @@ public function handle()
         $this->handle();
     }
 
-    public function processAllStreams()
+    public function processAllStreams(): void
     {
-        $streams = config('eventstore.streams');
+        if($this->option('persist') || (!$this->option('persist') && !$this->option('volatile'))) {
+            $this->connectToStream('eventstore.streams');
+        }
+
+        if($this->option('volatile')) {
```

Comment:
Wouldn't it make sense to do `if($this->option('volatile')) { ... } else { $this->connectToStream('eventstore.streams'); }`? Also - to refactor it per
```
if ($this->option('persist')
$this->connectToStream('eventstore.streams', function(EventStore $eventStore, string $stream){
$this->processPersist($eventStore, $stream);
})
```